draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -287,23 +286,23 @@ The use of the loss bits is negotiated using a transport parameter:
 
 loss_bits (0x1055):
 
-: The loss bits transport parameter is an integer value that can be set
-  to 0 or 1 indicating the level of QL bits support.
+: The loss bits transport parameter is an integer value, encoded as a
+  variable-length integer, that can be set to 0 or 1 indicating the level of
+  QL-bits support.
```

Comment:
Do you want to include the changes from PR #4 here?

Reply:
I do not think they are needed. "Supports loss bits extension" (from PR #4) is on its face obvious (how else would it know to send this TP?) and does not add anything else.